src/include/ipc4/base_fw.h

```diff
@@ -558,6 +558,21 @@ struct ipc4_system_time_info {
 	struct ipc4_system_time dsp_time;
 } __attribute__((packed, aligned(4)));
 
+struct ipc4_ext_system_time {
+	//! Lower DWORD of current system time value.
+	uint32_t utc_l;
+	//! Upper DWORD of current system time value.
+	uint32_t utc_u;
+	//! Lower DWORD of current RTC value.
+	uint32_t rtc_l;
+	//! Upper DWORD of current RTC value.
+	uint32_t rtc_u;
+	//! Lower DWORD of current HH value.
+	uint32_t hh_l;
```

Comment:
`hh` should not be used in a common interface definition as it is Intel HW specific name. `art` (always running timer) seems to be more appropriate.